README.md

```diff
@@ -126,15 +126,19 @@ tx.add_transfer(
     recipient="cosmos103l758ps7403sd9c0y8j6hrfw4xyl70j4mmwkf", amount=387000
 )
 tx.add_transfer(recipient="cosmos1lzumfk6xvwf9k9rk72mqtztv867xyem393um48", amount=123)
-pushable_tx = tx.get_pushable()
 
 
-# Optionally submit the transaction using your preferred method.
-# This example uses the httpx library.
-import httpx
+# Submit the transaction
+rpc_url = "https://rpc.cosmos.network/"
+
+# Method 1
+r = tx.broadcast(url=rpc_url)
```

Comment:
I don't really want this API or the `requests` dependency. Cosmospy is not a networking library.